modules/notifications/api.php

```diff
@@ -101,7 +101,13 @@ private static function check_group( $group ) {
 					$result = 'in' === $condition['operator'] ? $in_array : ! $in_array;
 					break;
 				case 'plugin':
-					$result = is_plugin_active( $condition['plugin'] );
+					$is_plugin_active = is_plugin_active( $condition['plugin'] );
```

Comment:
Fatal error: Uncaught Error: Call to undefined function Elementor\Modules\Notifications\is_plugin_active() in wp-content\plugins\elementor\modules\notifications\api.php:104